### PR TITLE
On board on-demand data exhaust APIs

### PR DESCRIPTION
**Pull request by sowmya-dixit**

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -4821,6 +4821,60 @@ kong_apis:
...
+    uris: "{{ data_service_prefix }}/v3/job/request/list"
+    upstream_url: "{{ analytics_api_service_url }}/job/request/list"
+    strip_uri: true
+    plugins:
+     - name: jwt
+     - name: cors
+     - "{{ statsd_pulgin }}"
+     - name: acl
+       config.whitelist:
+         - 'dataAccess'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:4870]: @sowmya-dixit  it should have dataAdmin ACL

File: ansible/roles/kong-api/defaults/main.yml [hunk 2]
```diff
@@ -4821,6 +4821,60 @@ kong_apis:
...
+    uris: "{{ data_service_prefix }}/v3/job/request/submit"
+    upstream_url: "{{ analytics_api_service_url }}/job/request/submit"
+    strip_uri: true
+    plugins:
+     - name: jwt
+     - name: cors
+     - "{{ statsd_pulgin }}"
+     - name: acl
+       config.whitelist:
+         - 'dataAccess'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:4834]: @sowmya-dixit  it should have dataCreate ACL

Comment (sowmya-dixit) [reply ansible/roles/kong-api/defaults/main.yml]: Updated ACL to dataCreate

File: ansible/roles/kong-api/defaults/main.yml (hunk 1 above)
Comment (sowmya-dixit) [reply ansible/roles/kong-api/defaults/main.yml:4870]: Keeping dataAccess as ACL as user can retrieve only his jobs

Comment (SMYALTAMASH) [reply ansible/roles/kong-api/defaults/main.yml:4870]: yes, then Access ACL makes sense

**Review by SMYALTAMASH — APPROVED** [review]

**Pull request merged**